.github/workflows/conformance-kind.yaml

```diff
@@ -98,6 +99,7 @@ jobs:
         with:
           version: ${{ env.kind_version }}
           config: ${{ env.kind_config }}
+          cluster_name: kind-cluster-ipv4
```

Comment:
you can template this

Reply:
see how I did in this job https://github.com/cilium/cilium/blob/main/.github/workflows/conformance-k8s-kind.yaml

Reply:
it should work if you just append `ipv6` here

https://github.com/cilium/cilium/blob/2c367de92d1825bf2ab80de2fd4e848ee2e28c46/.github/workflows/conformance-k8s-kind.yaml#L38-L39